schemas/keybindings.json

```diff
@@ -0,0 +1,389 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "definitions": {
+        "key": {
+            "type": "string",
+            "minLength": 1,
+            "not": {
+                "pattern": "---|([^-]+)-\\1"
```

Comment:
The issue with comprehensive regex that  it will be shown when wrong value is passed. And it will look unreadable and hard to figure out what's wrong. :) I think that the best option here is to implement simple plugin which checks format completely.

Reply:
The issue with comprehensive regex that  it will be shown when wrong value is passed. And it will look unreadable and hard to figure out what's wrong. :) I think that the best option here is to implement simple plugin which checks format completely.